test/06-extra-args/CMakeLists.txt

```diff
@@ -7,7 +7,66 @@ find_package(Pytest REQUIRED)
 enable_testing()
 
 pytest_discover_tests(
-    TestExtraArgs
+    TestExtraArgs.extraArgs
     EXTRA_ARGS "--capture=no" "--cmdopt=demo"
     DEPENDS test_extra_args.py
 )
+
+set(EXPECTED
+    "TestExtraArgs.extraArgs.test_requires_no_capture"
+    "TestExtraArgs.extraArgs.test_requires_args"
+)
+
+add_test(
+    NAME TestExtraArgs.Validate
```

Comment:
This test isn't essential since the discovered Python tests will fail if the arguments aren't passed. We're not testing how the tests are renamed, so I feel it's best to keep it simpler.